File: erpnext_extensions/petty_management/report/pm_opening_advance_availability_report/pm_opening_advance_availability_report.py
# Copyright (c) 2026, Farbod Siyahpoosh and contributors
# For license information, please see license.txt

from erpnext_extensions.petty_management.services.report_service import (
	get_pm_opening_advance_availability_report_data,
)


def execute(filters=None):
	return get_pm_opening_advance_availability_report_data(filters)

File: erpnext_extensions/petty_management/report/pm_opening_advance_availability_report/pm_opening_advance_availability_report.js
// Copyright (c) 2026, Farbod Siyahpoosh and contributors
// For license information, please see license.txt

frappe.query_reports["PM Opening Advance Availability Report"] = {
	filters: [
		{
			fieldname: "company",
			label: __("Company"),
			fieldtype: "Link",
			options: "Company",
		},
		{
			fieldname: "employee",
			label: __("Employee"),
			fieldtype: "Link",
			options: "Employee",
		},
		{
			fieldname: "holder",
			label: __("PM Holder"),
			fieldtype: "Link",
			options: "PM Holder",
		},
		{
			fieldname: "only_available",
			label: __("Only With Available Balance"),
			fieldtype: "Check",
		},
	],
};
